RELEASE/scripts/autoscend/quests/level_09.ash

```diff
@@ -989,7 +996,8 @@ boolean L9_highLandlord()
 	{
 		return false;
 	}
-	if(get_property("chasmBridgeProgress").to_int() < 30)
+	if(get_property("chasmBridgeProgress").to_int() < 30 && !auto_haveBatWings() ||
+		get_property("chasmBridgeProgress").to_int() < 25 && auto_haveBatWings())
```

Comment:
```suggestion
	if(get_property("chasmBridgeProgress").to_int() < (!auto_haveBatWings() ? 30 : 25))
```

Reply:
this condition needs parens, or you can simplify it like this

Reply:
actually there are a ton of places in this file that check less than 30 for lumber of fasteners, you may want to put the amount in a function so you can use it everywhere

Reply:
Good call. I'll go through and update those other spots